packages/google_sign_in/google_sign_in_web/lib/src/gapi.dart

```diff
@@ -0,0 +1,169 @@
+@JS()
```

Comment:
You should consider making this a `part` file and making pretty much everything in here private (with non-private `@JS` names) otherwise all of these classes will be part of the public API of `google_sign_in_web`

Reply:
_We discussed this offline and decided to keep this as an import from src instead of a part'd lib._